app/lib/submission_builder/state1099_r.rb

```diff
@@ -7,7 +7,7 @@ def document
       state_abbreviation = form1099r.intake.state_code.upcase
 
       build_xml_doc("IRS1099R", documentId: "IRS1099R-#{form1099r.id}") do |xml|
-        xml.PayerNameControlTxt form1099r.payer_name_control
+        xml.PayerNameControlTxt form1099r.payer_name_control.gsub(/\s+|-/, '').upcase[0..3]
```

Comment:
I checked the federal xsd, and this value we pass through should match the same validations in the State schema. The validation errors came from wrong test data - both in our fixtures, and it is possible to put bad data in the federal info controller.

I would strongly prefer to update our test fixtures to have correct values. It would also be nice to put a validation error for this field on the federal_info_controller, but not sure if that's needed or very straightforward to implement due to how it's structured.